src/terminal.rs

```diff
@@ -48,6 +48,8 @@ pub fn columns() -> u16 {
 enum Terminal {
     /// iTerm2
     ITerm2,
+    /// Terminology
```

Comment:
Can you perhaps add a link to the Terminology website?  I'll add one for the iterm website as well :)